src/hub.html

```diff
@@ -10,6 +10,7 @@
     <link rel="shortcut icon" type="image/png" href="/favicon.ico">
     <title>Get together | Hubs by Mozilla</title>
     <link href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,400i,700" rel="stylesheet">
```

Comment:
It looks like we've stopped using Zilla Slab completely. Might as well remove this font reference and cut the dead weight?